src/main/java/blackjack/domain/betting/Betting.java

```diff
@@ -0,0 +1,37 @@
+package blackjack.domain.betting;
+
+public class Betting {
+
+    private Double bettingMoney;
+
+    public Betting(String bettingMoney) {
+        this.bettingMoney = validateMoneyInput(bettingMoney);
+    }
+
+    public Betting calculateBettingMoney(Double profitRate) {
+        bettingMoney *= profitRate;
+        return this;
+    }
+
+    public Double getBettingMoney() {
+        return bettingMoney;
+    }
+
+    private double validateMoneyInput(String bettingMoney) {
+        double bettingMoneyValue;
+
+        try {
+            bettingMoneyValue = Double.parseDouble(bettingMoney);
```

Comment:
```suggestion
        try {
            double bettingMoneyValue = Double.parseDouble(bettingMoney);
```
따로 변수선언을 하신 이유가 있을까요?